_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -213,8 +213,30 @@ class HealthCareApp extends React.Component {
     this.setState({ applicationData: newApplicationData });
   }
 
+  resetNullValues(objectData) {
+    const context = this;
```

Comment:
I've been digging into this for awhile and can't seem to figure out where `this` went :( For now I will add a comment to return to this later and investigate.